apps/omg/config/config.exs

```diff
@@ -14,4 +14,9 @@ config :omg, :eip_712_domain,
   version: "1",
   salt: "fad5c7f626d80f9256ef01929f3beb96e058b8b4b0e3fe52d84f054c0e2a7a83"
 
+config :omg, OMG.Utils.Tracer,
+  service: :omg,
+  adapter: SpandexDatadog.Adapter,
+  env: System.get_env("APP_ENV")
```

Comment:
The config files are evaluated at compile time so it will not be reflected for compiled code.

Please consider [`deferred_config`](https://github.com/mrluc/deferred_config) e.g. as used in https://github.com/omisego/elixir-omg/blob/master/apps/omg_child_chain_rpc/config/dev.exs#L11-L12

Reply:
Great, thank you very much. I'll make that change. 👍